### ISSUE TITLE
dsonlint

### ISSUE THREAD

**Issue by JDeuce** [dogescript/DSON#24]
Hey man I made this a long time ago and forgot about it http://dsonlint.github.io/. Not sure if anyone else has done one. Might be cool to link it from the main pages, as I think it's a cool way to show off the spec.

**Comment by AnEmortalKid** [comment]
Yeah, definitely I think so too! Let me see if i can have a subdomain so `dson.dogescript.io` also points to it? 

Or would you want to contribute that to `dson.github.io` as a repo here? we have the dogescript one like that: https://github.com/dogescript/dogescript.github.io

**Comment by AnEmortalKid** [comment]
@vpzomtrrfrt where do you publish the dogeon.xyz from?

**Comment by vpzomtrrfrt** [comment]
I think it's hosted by GitHub Pages from this repo

**Comment by AnEmortalKid** [comment]
Aight, that's true^. 

@JDeuce mind making a PR on the html files with a link to your project above the "parser implementations section"?
